introduction.md

```diff
@@ -0,0 +1 @@
+My Name is Tomasz. I come from Poland and now I am learning about GitHub.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.